net/dnsproxy/files/dnsproxy.config

```diff
@@ -0,0 +1,42 @@
+
+# For documents, please see https://github.com/AdguardTeam/dnsproxy#usage
+
+config dnsproxy 'enabled'
+	option enabled '0'
+
+config dnsproxy 'config'
+	option listen_addr '127.0.0.1'
+	option listen_port '5353'
+	option log_file ''
+	option all_servers '0'
+	option fastest_addr '0'
+	option insecure '0'
+	option ipv6_disabled '0'
+	option max_go_routines ''
+	option rate_limit ''
+	option refuse_any '0'
+	option udp_buf_size ''
+	option verbose '0'
+
+config dnsproxy 'bogus_nxdomain'
+	list ip_addr ''
```

Comment:
This could be `list bogus_nxdomain` under `config`/`global` since there's just this one knob

Reply:
I think splitting this could be more intuitive and clean (list vs. option).

Reply:
You could of course have multiple 'list nxdomain' entries under 'global', which would also get rid of `ip_addr`, but this works too ;)